scripts/pum

```diff
@@ -249,7 +249,7 @@ class Pum:
             print(e)
             exit(1)
 
-    def run_upgrade(self, pg_service, table, delta_dirs, variables=None, max_version=None):
+    def run_upgrade(self, pg_service, table, delta_dirs, verbose, variables=None, max_version=None):
```

Comment:
I am tempted to add a commit that removes the default values for `variables` and `max_version` here and in `run_test_and_upgrade`. Because they are always specified when calling `run_upgrade` and `run_test_and_upgrade`.

Reply:
I would not modify the API as it might be used directly in Python (instead of command line).
What would be the interest of removing the default values?

Reply:
I did not think Pum was used as a library.  And if it is I don't what the API boundary is.  The run_ functions?  If the run_ functions are the API I should add verbose as the last argument, with a default value.

And if we decide that we can change the run_ function signatures the way we want I think it makes sense to remove the default values as they are not currently used, the arguments being always specified by the calling code.